PizzaBuilder/PizzaBuilder/Program.cs

```diff
@@ -0,0 +1,229 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace PizzaBuilder
+{
+    public interface IBuilder
+    {
+        void BulidDough();
+        void BuildSauce();
+        void BuildTopping();
+    }
+    public class WhitePizzaBulider : IBuilder
+    {
+        private WhitePizza _product = new WhitePizza();
+        public WhitePizzaBulider()
+        {
+            this.Reset();
+        }
+        public void Reset()
+        {
+            this._product = new WhitePizza();
+        }
+        public void BulidDough()
+        {
+            this._product.Add("White Dough");
+        }
+        public void BuildSauce()
```

Comment:
It would be nice if you specified some sauces like (White sauce, red sauce) or what you want rather than just writing the word Sauce.